omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java

```diff
@@ -30,10 +31,26 @@ public void controller(FragmentConfiguration config,
 			@SpringBean("adtService") AdtService adtService,
 			@SpringBean("visitService") VisitService visitService,
 			@SpringBean("visitTypeHelper") VisitTypeHelper visitTypeHelper,
-			UiSessionContext sessionContext){
+			UiSessionContext sessionContext,
+			HttpServletRequest request,
+			@SpringBean("patientService")PatientService patientService){
 
-		config.require("patient");
 		Object patient = config.get("patient");
```

Comment:
I'm not 100% sure, but I think the problem is that the patient *is* required here, and was previously getting passed through via some other means.

See my comment on RA-1273 regarding the patient merge push--can you test if this change fixes that?